65black/so-lite-react
# #162832580 Add Voting Functionality

#### What does this PR do?
This PR adds the ability for a logged in user to upvote or downvote an answer by clicking on a button.

#### Description of Task to be completed?
- add tests for new feature
- convert QuestionAnswer component to class component

#### How should this be manually tested?
- Clone the repo, run `npm install` and run `npm start`
- Log in to your account #3 
- Navigate to `localhost:7000/questions` and click on any question
- Choose any answer and click on the upvote or downvote button
- The upvote or downvote count should increase by one

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#162832580

#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2018-12-28 at 9 15 35 pm" src="https://user-images.githubusercontent.com/36507972/50527010-cdea8c80-0ae5-11e9-9b06-200de071582b.png">
<img width="1440" alt="screenshot 2018-12-28 at 9 15 11 pm" src="https://user-images.githubusercontent.com/36507972/50527011-cdea8c80-0ae5-11e9-8b89-4513948f9b26.png">

#### Questions:
N/A
